source/2019-12-13-ember-3-15-released.md

```diff
@@ -0,0 +1,127 @@
+any bugs before they are published as a final release in six weeks' time.
+
+You can read more about Ember's general release process at the [Release Dashboard](http://emberjs.com/builds/).
+
+---
+
+## Other Changes in Ember.js 3.15
+
+Deprecations are added to Ember.js when an API will be removed at a later date. Each deprecation has an entry in the deprecation guide describing the migration path to a more stable API. Deprecated public APIs are not removed until a major release of the framework.
+
+Consider using the [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow) addon if you would like to upgrade your application without immediately addressing deprecations.
+
+For more details on changes in Ember.js 3.15, please review the [Ember.js 3.15.0 release page](https://github.com/emberjs/ember.js/releases/tag/v3.15.0).
+
+**Deprecate `Component#isVisible`**
+
+Classic Ember components support using the `isVisible` property to toggle the
+visibility of a component's root element. This API was very common in early
+Ember 1.x applications but is rare today. In practice use of template bindings
+has supplanted use of the `isVisible` API.
+
+`isVisible` is deprecated as of Ember 3.15. 
+
+To transition away from this, we recommend using conditionals in a template:
+
+```handlebars
+{{#if showComponent}}
+  {{component}}
+{{/if}}
+
+{{! or }}
+<div hidden={{isHidden}}></div>
+
+```
+
+See the [deprecation guide for
+`isVisible`](https://deprecations.emberjs.com/v3.x#toc_ember-component-is-visible)
+for more information.
+
+**Deprecate `{{partial}}`** 
+
+As of the introduction of [template-only
+components](https://api.emberjs.com/ember/3.15/classes/Component)
+and [Glimmer
+components](https://api.emberjs.com/ember/3.15/modules/@glimmer%2Fcomponent),
+as well as of the
+[template-only-glimmer-components](https://guides.emberjs.com/release/configuring-ember/optional-features/#toc_template-only-glimmer-components)
+optional feature the `{{partial}}` API in Ember no longer provides any unique
+benefits. Instead it has several downside including difficult to teach scoping 
+rules and poor performance.
+
+See the [deprecation guide for
+`{{partial}}`](https://deprecations.emberjs.com/v3.x#toc_ember-partial) for
+an example of how to migration existing use to component APIs.
+
+**Deprecate intimate `window.ENV` API** 
+
+Ember applications long ago shifted to using `window.EmberENV` as the home of
+various configuration options. As of 3.15 application code which uses `window.ENV`
+for Ember configuration values will cause a deprecation warning to be issued.
+Support for this intimate API will be removed in 3.17.0.
+
+See [emberjs/ember.js#18441](https://github.com/emberjs/ember.js/pull/18441)
+for more details.
+
+## Ember Data
+
+There were two new features and no deprecations for Ember Data v.3.15.0.
+
+### IDENTIFIERS turned on (1 of 2)
+
+(see https://github.com/emberjs/data/pull/6366) 
```

Comment:
@runspired @igorT this feature PR had no description. Please provide one in place of this line.